pkg/js/helpers.js

```diff
@@ -658,6 +659,10 @@ function SPF_BUILDER(value) {
     if (!value.raw) {
         value.raw = '_rawspf';
     }
+    if (!value.ttl) {
+        value.ttl = defaultArgs['defaultTTL'];
```

Comment:
I think we should just not set a ttl on the final txt record if ttl is not set. That feels more consistent.

Reply:
If I leave `value.ttl` undefined (in case not specified in `dnsconfig.js`), it fails because of:
```
ERROR: Unknown modifier type
```

That's why I thought simply using the defaultTTL (which will be used anyway) will be the best way. The other solution might be wrapping a `IF-Else` around both `r.push(TXT(...))` lines.

Reply:
I think its probably best to have the if/else blocks tbh. we generally prefer letting the default ttls get bound as late as possible.

Reply:
Understood. Changed!